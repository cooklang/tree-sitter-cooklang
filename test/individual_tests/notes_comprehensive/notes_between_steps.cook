First step here.

> Important reminder!

Second step here.