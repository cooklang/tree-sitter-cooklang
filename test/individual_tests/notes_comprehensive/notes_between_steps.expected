(recipe
  (step
    (plain_text))
  (note)
  (step
    (plain_text)))